src/remoteCollection/collect.js

```diff
@@ -135,7 +135,13 @@ function sendRemoteRequest(generator) {
         } // shouldRequestNewToken
 
         debug('sendRemoteRequest returned error %O', err);
-        generator.res = err;
+        if (get(generator, 'connection.simple_oauth')) {
+          const method = generator.connection.simple_oauth.method;
+          generator.res =
+            new Error(`simple-oauth2 (method=${method}): ${err.message}`);
```

Comment:
Why did you add this here as well? Should it be different from the one below?

Reply:
This error is still an authentication error, based on the simple-oauth2 configuration, but the error here was an authentication error on the remote data source, while the other one was an error actually calling simple oauth's getToken function